src/lib/driver/command.ts

```diff
@@ -0,0 +1,6 @@
+export enum DriverCommand {
+  updateLogConfig = "driver.update_log_config",
+  getLogConfig = "driver.get_log_config",
+  enableStatistics = "driver.enable_statistics",
+  disableStatistics = "driver.disable_statistics",
```

Comment:
Do we want to have this as 2 methods, or 1 `set_statistics` method or even have 1 `set_config` method ?

Reply:
I don't think we gain much by having a single `set_config` method because you will set different parts of the config at different times. Changing the log level will be random, enabling statistics will be when the client connects to the server if the user opted in or when the user opts in, disabling statistics will be when the user opts out.

As to one command or two, I don't have a preference other than the fact that the commands have a clean relationship with the API methods in the `zwave-js` docs if they remain separate

Reply:
following ZJS Methods makes sense, since we follow their structure too.